torch/__init__.py

```diff
@@ -1364,6 +1364,7 @@ def _assert(condition, message):
 # side effect of adding to the imported module's members for other users.
 from torch import cuda as cuda
 from torch import cpu as cpu
+from torch import xla as xla
```

Comment:
I don't think we want this - why do you need it? This will have `torch.xla` show up by default as an available module when you `import torch`. But that doesn't really make sense for anyone not using `torch_xla`.